app/api.py

```diff
@@ -34,6 +32,7 @@
 import uuid
 
 import django.utils.html
+from django.utils.translation import activate
```

Comment:
I didn't know it, but looks like it is discouraged to directly import a specific function like this:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports
You should instead do:
```python
from django.utils import translation
```

That style makes sense to me. It is hard to understand what is activated by just reading `activate(query_lang)`.

IMHO one line below here can be considered an exception because it's a common pattern.

Reply:
Ok, done. There's other places where we import functions, but I wouldn't want to make it worse.